examples/cd_clf_cifar10.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<p>I would probably import the <code>f1_score</code>  here as then it's in context, otherwise have to scroll to the top to check where it comes from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>